view/adminhtml/templates/configuration.phtml

```diff
@@ -0,0 +1,149 @@
+
+			var fixHelper = function(e, ui) {
+				ui.children().each(function() {
+					$(this).width($(this).width());
+				});
+
+				return ui;
+			};
+
+			setTimeout(function () {
+				$('form[action*="algoliasearch"] .admin__control-table tbody').sortable({
+					containment: "parent",
+					items: 'tr',
+					tolerance: 'pointer',
+					helper: fixHelper,
+					start: function (event, ui) {
+						$(ui.item).css('margin-left', '10px');
+					}
+				});
+
+				$('form[action*="algoliasearch"] .admin__control-table tbody tr td').css('cursor', 'move');
+			}, 1000);
+
+
+			var url = window.location.href;
+			if (url.indexOf('section/algoliasearch_') !== -1) {
+
+				// Hide click analytics config if not available
+				if (isClickAnalyticsEnabled === false) {
+					$('#algoliasearch_cc_analytics_cc_analytics_group table').hide();
+				}
+
+				// Adding links and videos to the config panel
+                $('.algolia-admin-content').html(linksAndVideoTemplate);
+
+            	// New notice management
+            	extensionNotices.each(function(notice){
+            		if (notice.method === 'before') {
+        				$(notice.selector).before(notice.message);
+        			} else if (notice.method === 'append') {
+						$(notice.selector).append(notice.message);
+        			} else if (notice.method === 'after') {
+						$(notice.selector).after(notice.message);
+        			}
+            	});
+			}
+
+			// SYNONYMS
+
+			handleSynonyms($('#algoliasearch_synonyms_synonyms_group_enable_synonyms').val());
+
+			$(document).on('change', '#algoliasearch_synonyms_synonyms_group_enable_synonyms', function() {
+				handleSynonyms($(this).val());
+			});
+
+			function handleSynonyms(enabled) {
+				var $rows = $('#row_algoliasearch_synonyms_synonyms_group_synonyms, #row_algoliasearch_synonyms_synonyms_group_oneway_synonyms, #row_algoliasearch_synonyms_synonyms_group_synonyms_file');
+
+				if (enabled === '1') {
+					$rows.show();
+				} else {
+					$rows.hide();
+				}
+			}
+
+			// ORDERED / UNORDERED
+			var $attributesRows = $('#algoliasearch_products_products_product_additional_attributes, #algoliasearch_categories_categories_category_additional_attributes');
+			initAttributes($attributesRows);
+
+			$attributesRows.on('click', 'button[id^="addToEndBtn"]', function (e) {
+				initAttributes($attributesRows);
+			});
+
+			$attributesRows.on('change', 'select[name$="[searchable]"]', function (e) {
+				handleAttributes($(this));
+			});
+
+			function initAttributes($attributesRows) {
+				$attributesRows.find('select[name$="[searchable]"]').each(function(e) {
+					handleAttributes($(this));
+				});
+			}
+
+			function handleAttributes($selectBox) {
+				var selectValue = $selectBox.val(),
+					$input = $selectBox.parent('td').next().find('select[name$="[order]"]');
+
+				if(selectValue !== '1') {
+					$input.hide();
+				}
+				else {
+					$input.show();
+				}
+			}
+
+			// EVENT TRACKING
+            algoliaEventsTracking.track();
```

Comment:
This needs to go to the common file so that we track all events. It might be best to put it with tracking.phtml just after the object if not common.